lib/samson/form_builder.rb

```diff
@@ -36,7 +36,11 @@ def input(
           end
         else
           input_html[:class] ||= "form-control"
-          content = label(attribute, label, class: "col-lg-2 control-label")
+          content = if label.presence
+            label(attribute, label, class: 'col-lg-2 control-label')
+          else
+            content_tag(:div, '', class: 'col-lg-2 control-label')
```

Comment:
why is an empty label bad ? (addd comment for that)

Reply:
is is it different from this:
`label(attribute, '', class: 'col-lg-2 control-label')`

Reply:
how is it different from a label with an empty value ?